apps/admin/frontend/test/helpers/mock_api_client.ts

```diff
@@ -154,49 +155,60 @@ export function createApiMock(
 
     async authenticateAsVendor() {
       // First verify that we're logged out
-      await screen.findByText('VxAdmin Locked');
+      await vi.waitFor(() => screen.getByText('VxAdmin Locked'));
```

Comment:
Is this file intentionally using `vi.waitFor` instead of `screen.findBy*`?

Reply:
Fixed.